src/test/scripts/functions/unary/frame/input/Adult_csv/adult.csv.mtd

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
As discussed remove the adult dataset from this PR and emulate it with reasonably small random data.

Reply:
Test input files have been removed from the PR.